content/docs/capabilities/ppl.mdx

```diff
@@ -322,7 +322,7 @@ allow:
     - record:
         type: pomerium.io/ExternalDataSource
         field: id
```

Comment:
`id` field is not a list 

I believe `has` is only applicable to `foreign_field`. 

@calebdoxsey do you have a good example maybe for `has` usage in `record` ?

Reply:
```yaml
allow:
  and:
  - record:
      type: example.com/geoip
      field: country
      is: "US"
  - record:
      type: example.com/hr_user
      field: departments
      has: "engineering"
```

Reply:
@calebdoxsey  Is this part necessary?  Or is this just showing another operator that also uses list string matching?
```yaml
  - record:
      type: example.com/geoip
      field: country
      is: "US"
```

Reply:
Right that's showing using `is`. The record operator supports `contains`, `ends_with`, `is`, `starts_with` and `has`. It depends on what data is stored. If it's a list you'd want to use `has`. If it's a string you'd want to use one of the string matchers.